tensorboard/plugins/graph/polymer3/tf_graph_controls/tf-graph-controls.ts

```diff
@@ -12,563 +12,1408 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
-namespace tf.graph.controls {
-  interface DeviceNameExclude {
-    regex: RegExp;
-  }
 
-  const DEVICE_NAME_REGEX = /device:([^:]+:[0-9]+)$/;
+import {PolymerElement, html} from '@polymer/polymer';
+import {computed, customElement, property} from '@polymer/decorators';
+import * as _ from 'lodash';
+import '@polymer/iron-icon';
+import '@polymer/paper-button';
+import '@polymer/paper-dropdown-menu/paper-dropdown-menu';
+import '@polymer/paper-icon-button';
+import '@polymer/paper-item';
+import '@polymer/paper-listbox';
+import '@polymer/paper-radio-group';
+import '@polymer/paper-toggle-button';
+import '@polymer/paper-tooltip';
```

Comment:
use irons_and_papers. Audit the whole directory by searching for `@polymer/(iron|paper)-`